src/game.cpp

```diff
@@ -11659,13 +11659,29 @@ void game::pldrive(int x, int y)
         }
     }
 
+    int turn_delta = 15 * x;
+    if (turn_delta != 0) {
```

Comment:
I'm fighting a decade's worth of muscle memory here; a makefile target to restyle the code to the right form would help a lot.